### PR TITLE
Pbeckwith/cpm k8 update

### PR DESCRIPTION
**Pull request by Philip-R-Beckwith**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

CPM Release notes.
Adding a callout for max script log length. 

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx [hunk 1]
```diff
@@ -253,6 +253,10 @@ To avoid these issues, set wait conditions that will pause the script until the
 
 You can also manually log monitor results to the [script log](/docs/synthetics/new-relic-synthetics/using-monitors/viewing-monitor-results#viewing-logs). Use logging to troubleshoot a script: to discover which step of your script is failing, include a log function along with each key step in your script.
 
+<Callout variant="important">
+  The Maximum Log length is 50000 Bytes. Script logs larger than 50000 Bytes are truncated.
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx:257]:
```suggestion
  The maximum log length is 50000 Bytes. Script logs larger than 50000 Bytes are truncated.
```

Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx:257]: This seems like a capitalization error, but can you confirm real quick before I merge? Thanks!

**Review by bradleycamacho — APPROVED** [review]

File: src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx (hunk 1 above)
Comment (Philip-R-Beckwith) [reply src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx:257]: Yup it was.

**Review by urbiz-nr — APPROVED** [review]
Thanks for the updates, @Philip-R-Beckwith!

**Pull request merged**